### PR TITLE
updates upload box styling and fixes edit view xplorer

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- updates upload box styling that is looks same as in filer
- removes position relative from filer-dropzone content to show image in xplorer edit mode

### CODE REVIEW

**Comment by Brankub** [comment]
:+1:

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css [hunk 1]
```diff
@@ -203,13 +201,16 @@
     padding: 15px;
     border-radius: 3px;
     background: #fff;
-    box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.7);
+    box-shadow: 0 0 5px 0 rgba(0,0,0,0.2);
 }
-.filer-dropzone-info-message .icon {
-    font-size: 30px;
+.filer-dropzone-info-message .icon,
```
Comment (FinalAngel) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: what would happen if I have an image with a class "icon"? Specificity might here be a better solution?

File: aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/image.html [hunk 2]
```diff
@@ -66,7 +66,7 @@
                 </div>
                 <div class="filer-dropzone-error-message js-filer-dropzone-error-message hidden">
                     <div class="icon"><span class="fa fa-cloud-upload"></span></div>
-                    <div class="js-filer-dropzone-upload-accept">
+                    <div class="js-filer-dropzone-upload-accept text">
```
Comment (FinalAngel) [inline aldryn_bootstrap3/templates/aldryn_bootstrap3/plugins/image.html]: text is very very generic, that might clash with other frontend frameworks or?

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css [hunk 3]
```diff
@@ -203,13 +201,16 @@
...
+.filer-dropzone-error-message .icon {
+    font-size: 35px;
+    color: #0bf;
 }
-.filer-dropzone-info-message .text {
-    margin: 15px 0 10px 0;
+.filer-dropzone-info-message .text,
+.filer-dropzone-error-message .text {
+    margin: 5px 0 10px 0;
 }
```
Comment (sergii-kulakovskyi) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: suggest to optimize selectors and omit last zero:
```
.filer-dropzone-info-message,
.filer-dropzone-error-message {
    .text {
        margin: 5px 0 10px;
    }
}
```

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css (hunk 1 above)
Comment (lory87) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: it will never happen because this icon is used in info message box which appears when we upload an image. It doesn't make sense to show images in this box :)

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css (hunk 3 above)
Comment (sergii-kulakovskyi) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: or even create new class `.filer-dropzone-message`

Comment (lory87) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: unfortunately it is not sass ;)

**Comment by lory87** [comment]
@FinalAngel renamed icon and text class names

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css (hunk 3 above)
Comment (sergii-kulakovskyi) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: oops )

**Comment by FinalAngel** [comment]
thank you @lory87 LGTM :+1:

File: aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css [hunk 4]
```diff
@@ -191,11 +188,12 @@
 .filer-dropzone-error-message {
     position: fixed;
     left: 50%;
-    bottom: 0;
+    top: 50%;
```
Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: what does this do? i remember we put the box with upload info specifically to the bottom of the page with Matthias

Comment (lory87) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: didn't know that. I centered the box vertically. I thought this was a mistake that the box was positioned bottom. Should I revert this change?

Comment (vxsx) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: yes please
the idea here is that it shouldn't overlap the image if the image is big and in the center of your screen

Comment (lory87) [inline aldryn_bootstrap3/static/aldryn_bootstrap3/css/base.css]: done

**Comment by vxsx** [comment]
:+1:
